gen/edu/uc/emiller/carapp/dao/ICarDAO.kt

```diff
@@ -0,0 +1,22 @@
+package edu.uc.emiller.carapp.dao
+
+import androidx.lifecycle.LiveData
+import androidx.room.*
+import edu.uc.emiller.carapp.dto.Car
+
+@Dao
+interface ICarDAO {
+
+    @Query("SELECT * FROM car")
+    fun getAllCars() : LiveData<List<Car>>
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    fun insertAll(cars : ArrayList<Car>)
+
+    @Insert
+    fun save(car : Car)
+
+    @Delete
+    fun delete(car : Car)
+
+}
```

Comment:
Basic layout for a DAO configured for use with Room DB. Almost identical to the prof's repo.